src/OpenTelemetry/Metrics/BaseExportingMetricReader.cs

```diff
@@ -91,7 +91,18 @@ internal override bool ProcessMetrics(in Batch<Metric> metrics, int timeoutMilli
             // TODO: Do we need to consider timeout here?
             try
             {
-                return this.exporter.Export(metrics) == ExportResult.Success;
+                OpenTelemetrySdkEventSource.Log.MetricReaderEvent("BaseExportingMetricReader calling Exporter.Export method.");
```

Comment:
it will get harder and harder with more logs.. Okay with this idea now, to get some visibility if things go wrong. We need to revisit and see how to improve the logs after gathering more feedback.